cmake/IgnConfigureProject.cmake

```diff
@@ -4,13 +4,13 @@
 #
 # ign_configure_project([VERSION_SUFFIX <pre|alpha|beta|etc>])
 #
-# Sets up an ignition library project.
+# Sets up an gazebo library project.
 #
 # NO_IGNITION_PREFIX: Optional. Don't use ignition as prefix in
 #     cmake project name.
 # REPLACE_IGNITION_INCLUDE_PATH: Optional. Specify include folder
 #     names to replace the default value of
-#     ignition/${IGN_DESIGNATION}
+#     gazebo/${GZ_DESIGNATION}
```

Comment:
We shouldn't change the include paths in stable releases because that's not backwards compatible

```suggestion
#     ignition/${GZ_DESIGNATION}
```